vhost/src/vhost_user/gpu_backend_req.rs

```diff
@@ -156,6 +159,18 @@ impl GpuBackend {
             .map_err(|e| io::Error::new(io::ErrorKind::Other, format!("{}", e)))
     }
 
+    /// Forward 2d gpu command SetScanout requests to the frontend.
+    pub fn set_dma_scanout(&mut self, scanout: &VhostUserGpuDMABUFScanout) -> io::Result<()> {
```

Comment:
I would rather name it `set_dmabuf_scanout` - to make it closer to the original name (or just `dmabuf_scanout`? but that seems weird).